featuremanagement/aio/_featuremanager.py

```diff
@@ -21,14 +22,14 @@ class FeatureManager(SyncFeatureManager):
     Feature Manager that determines if a feature flag is enabled for the given context
 
     :param configuration: Configuration object
-    :type configuration: dict
+    :type configuration: Mapping
     :keyword feature_filters: Custom filters to be used for evaluating feature flags
     :paramtype feature_filters: list[FeatureFilter]
     """
 
     def __init__(self, configuration, **kwargs):
         self._filters = {}
-        if configuration is None or not isinstance(configuration, dict):
```

Comment:
Mapping change shouldn't be a part of this PR- but I recall why this is the case so it's fine with me.

Reply:
The issue was that we missed updating async in the other PR and I didn't want it to get lost.